docs/reference/api/caching.md

```diff
@@ -0,0 +1,6 @@
+# Caching
+
+## Express Cache on Demand
+
+Implementation of [express-cache-on-demand](https://www.npmjs.com/package/express-cache-on-demand). It allows for similar and concurrent requests to compute the result only once. If multiple users request the same route at the same time, we load the content once, and send it to users instead of loading it for each request. It works for pages and pieces get requests when no user or session are involved.
```

Comment:
Please capitalize GET in this context and clarify that there are two things being cached:

1. Ordinary page requests (like a visit to the home page), for logged-out site visitors only, and only when `req.session` does not contain anything.
2. GET REST API requests for pages or pieces, for logged-out site visitors only, and only when `req.session` does not contain anything.

Otherwise this page is a good start, we will add more information here when the forthcoming caching tickets get implemented.